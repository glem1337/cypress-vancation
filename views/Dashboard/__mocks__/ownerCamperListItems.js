export default [
  {
    id: '0a91a2b1-3501-4c71-ae07-eae3d280f861',
    img: 'https://vancation-api-staging-assets.s3.us-west-1.amazonaws.com/store/camperphoto/54b394d3-d9ef-4698-acb0-0799932f1add/photo/585ec167567095de53f90cbcd0eb31b3.jpg',
    label: 'camper first',
    subtitle: '9544432',
  },
  {
    id: '22b70626-9334-4855-8ee4-d0e5d1d0192b',
    img: '/images/camper-default.jpg',
    label: 'Volkswagen California 6.1 4x4 VW Bus in Los Angeles, California, United States',
    subtitle: '1147333',
  },
];
