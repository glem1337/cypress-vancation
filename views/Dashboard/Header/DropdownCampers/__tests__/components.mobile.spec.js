import React from 'react';
import { shallow } from 'enzyme';

import ownerCamperListItems from 'views/Dashboard/__mocks__/ownerCamperListItems';
import ListCampersMobile from '../component.mobile';

describe('ListCampersMobile component tests', () => {
  const props = {
    handlerClick: jest.fn(),
    items: ownerCamperListItems,
    selectedCamper: ownerCamperListItems[0],
    activeKey: [],
    onCollapseChangeHandler: jest.fn(),
  };

  const component = shallow(<ListCampersMobile {...props} />);

  it('matches snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
