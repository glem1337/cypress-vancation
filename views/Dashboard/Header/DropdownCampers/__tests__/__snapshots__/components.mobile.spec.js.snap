// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListCampersMobile component tests matches snapshot 1`] = `
<Collapse
  activeKey={Array []}
  className="master-view__menu"
  ghost={true}
  onChange={[MockFunction]}
>
  <CollapsePanel
    header={
      <Item
        id="0a91a2b1-3501-4c71-ae07-eae3d280f861"
        img="https://vancation-api-staging-assets.s3.us-west-1.amazonaws.com/store/camperphoto/54b394d3-d9ef-4698-acb0-0799932f1add/photo/585ec167567095de53f90cbcd0eb31b3.jpg"
        label="camper first"
        showArrow={true}
        subtitle="9544432"
      />
    }
    key="uuid/v4"
    showArrow={false}
  >
    <ul
      className="master-view__menu-list"
    >
      <Item
        id="0a91a2b1-3501-4c71-ae07-eae3d280f861"
        img="https://vancation-api-staging-assets.s3.us-west-1.amazonaws.com/store/camperphoto/54b394d3-d9ef-4698-acb0-0799932f1add/photo/585ec167567095de53f90cbcd0eb31b3.jpg"
        key="0a91a2b1-3501-4c71-ae07-eae3d280f861"
        label="camper first"
        showArrow={false}
        subtitle="9544432"
      />
      <Item
        id="22b70626-9334-4855-8ee4-d0e5d1d0192b"
        img="/images/camper-default.jpg"
        key="22b70626-9334-4855-8ee4-d0e5d1d0192b"
        label="Volkswagen California 6.1 4x4 VW Bus in Los Angeles, California, United States"
        showArrow={false}
        subtitle="1147333"
      />
    </ul>
  </CollapsePanel>
</Collapse>
`;
